web/client/epics/layerdownload.js

```diff
@@ -97,7 +97,10 @@ const DOWNLOAD_FORMATS_LOOKUP = {
 };
 
 const { getFeature: getFilterFeature, query, sortBy, propertyName } = requestBuilder({ wfsVersion: "1.1.0" });
-
+const getCQLFilterFromLayer = (layer = {}) => {
```

Comment:
- no jsdoc
- can this be moved into a utils file?

Reply:
I can document but I think It is confusing documenting private and self explaining functions that are used inline in a single file.
For the moment I should not export this. It is used only here.